addons/account/models/account_journal.py

```diff
@@ -260,15 +260,19 @@ def _compute_available_payment_method_ids(self):
                     if vals['mode'] == 'unique' and (already_used or is_protected):
                         continue
 
-                    # Only the manual payment method can be used multiple time on a single journal.
-                    if payment_method.code != "manual" and already_used:
+                    # Payment method can be used multiple time on a single journal.
+                    if payment_method.code not in self._get_reusable_payment_methods() and already_used:
                         continue
 
                     pay_method_ids_commands_x_journal[journal].append(Command.link(payment_method.id))
 
         for journal, pay_method_ids_commands in pay_method_ids_commands_x_journal.items():
             journal.available_payment_method_ids = pay_method_ids_commands
 
+    @api.model
+    def _get_reusable_payment_methods(self):
+        return ['manual']
```

Comment:
```suggestion
        return {'manual'}
```
As this will only be used to check if the code is contained in it, using a set is faster.